frontend/actions/auth.js

```diff
@@ -134,12 +134,13 @@ export const logInWithGoogle = googleToken => {
     }
 };
 
-export const register = (login, password) => {
+export const register = (login, password, passwordRepeat) => {
     return async dispatch => {
         console.log('registering...');
 
         const name = login;
         password = SHA256(password).toString();
+        passwordRepeat = SHA256(passwordRepeat).toString();
```

Comment:
sprawdzenie powinno odbyć się na tym poziomie i jeśli hasła nie są identyczne wyświetlić jako podpowiedź. Wysyłanie takiej informacji do serwera, nie potrzebnie go by obicążyło koniecznością sprawdzania kolejnych warunków. Tak zamiast jednego zapytania podczas rejestracji, mamy ich 1+n gdzie n to ilość nieudanych prób wpisania takiego samego hasła

Reply:
Password checking moved here from _AuthEndpoint.kt_